CMakeLists.txt

```diff
@@ -57,6 +57,7 @@ set(CMAKE_RUNTIME_OUTPUT_DIRECTORY ${CMAKE_BINARY_DIR}/bin)
 set(CMAKE_Swift_MODULE_DIRECTORY ${CMAKE_BINARY_DIR}/swift)
 
 option(BUILD_SHARED_LIBS "build shared libraries" ON)
+option(BUILD_FULLY_STATIC "build fully static" NO)
```

Comment:
I think that we should prefer this to be spelt `FOUNDATION_FULLY_STATIC`. `BUILD_SHARED_LIBS` is a standard CMake option that we are changing the default value for.

Reply:
If we're going to make that change, we should do so separately; this is already merged in other branches and is just a cherry pick.

Reply:
(Changing that will also require a matching change to the scripts in the `swift-docker` repo too.)